client/src/main/java/tw/waterball/survive/io/client/utils/PlayAudio.java

```diff
@@ -0,0 +1,38 @@
+package tw.waterball.survive.io.client.utils;
+
+import javax.sound.sampled.*;
+import java.io.File;
+import java.text.NumberFormat;
+
+
+/**
+ * @author - c11037at@gmail.com
+ */
+public class PlayAudio {
+
+    //優化中
```

Comment:
add a todo comments that `TODO maybe we can cache each audio to prevent reloading each file, for example: how to resolve continuous shooting?`